### PR TITLE
Transaction DCC

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

File: spec/definitions/Transaction.yaml [hunk 1]
```diff
@@ -134,6 +134,43 @@ properties:
...
+        type: object
+        description: Quote amount and currency
+        properties:
+          amount:
+            type: number
+            format: double
+          currency:
+            type: string
+      usdMarkup:
+        description: TODO
```
Comment (adamaltman) [inline spec/definitions/Transaction.yaml]: description: `The amount of markup translated to USD.`

File: spec/definitions/Transaction.yaml [hunk 2]
```diff
@@ -134,6 +134,43 @@ properties:
...
+    type: boolean
+    readOnly: true
+  dcc:
+    description: Dcc detailed information. Can be null if hasDcc is false
+    type: object
+    readOnly: true
+    properties:
+      base:
+        type: object
+        description: Base amount and currency
```
Comment (2e3s) [inline spec/definitions/Transaction.yaml]: Although this is a normal domain language, I suggest to make the description clearer for anyone:
base - initial amount and currency to convert from
quote - suggested amount and currency to convert to

File: spec/definitions/Transaction.yaml [hunk 3]
```diff
@@ -134,6 +134,43 @@ properties:
     description: Acquirer name, available only when method is 'payment_card', else null
     type: string
     readOnly: true
+  hasDcc:
+    description: True if transaction has DCC applied
+    type: boolean
+    readOnly: true
+  dcc:
+    description: Dcc detailed information. Can be null if hasDcc is false
```
Comment (2e3s) [inline spec/definitions/Transaction.yaml]: I believe `Dynamic Currency Conversion` is better to use in the descriptions than `dcc` which is an abbreviation so should be in capital: `DCC`.
Also the next sentence would be `null if hasDcc is false` (more correct language  I think)

File: spec/definitions/Transaction.yaml (hunk 1 above)
Comment (XaosSintez) [inline spec/definitions/Transaction.yaml]: @adamaltman thanks, fixed

File: spec/definitions/Transaction.yaml [hunk 4]
```diff
@@ -134,6 +134,43 @@ properties:
...
+        description: Quote amount and currency
+        properties:
+          amount:
+            type: number
+            format: double
+          currency:
+            type: string
+      usdMarkup:
+        description: The amount of markup translated to USD
+        type: integer
```
Comment (2e3s) [inline spec/definitions/Transaction.yaml]: float

Comment (XaosSintez) [inline spec/definitions/Transaction.yaml]: @2e3s okay, thank you for review!

**Pull request merged**